src/components/Table/Table.css

```diff
@@ -77,6 +77,14 @@
 	background-color: var(--color--gray--lighter);
 }
 
+.table .clickable:not(.active):hover {
+	background-color: var(--color--gray--lighter);
+}
```

Comment:
Ah you did good but this rule was meant to *replace* the one above it. Sorry, I should have been more clear!

So basically you need to remove lines 76–79 now.

Reply:
ah that was mindless of me! should be remedied now